site/src/pages/tutorials/grpc/blog/implement-create.mdx

```diff
@@ -114,11 +113,6 @@ Now, let's make a call from the client. Note the code is minimal to keep things
   ```bash
   [main] INFO  e.a.client.blog.grpc.BlogClient - [Create response] Title: Another blog post Content: Creating a post via createBlogPost().
   ```
-  You can check that indeed two posts have been created by checking the server side.
-  ```bash
-  [armeria-common-worker-nio-2-1] INFO  e.a.server.blog.grpc.ServerMain - Created at 0 - My first blog
-  [armeria-common-worker-nio-2-1] INFO  e.a.server.blog.grpc.ServerMain - Created at 1 - Another blog post
-  ```
```

Comment:
Question: Why are these logs deleted? Maybe the original author intentionally left it to check the process from it.

Reply:
This change is related to Motivation & Modification 4.

The log above does not appear because there is no logging code in BlogService.java or Main.java. Therefore, I remove those logs.

Alternatively, It can be resolved by adding the following code `logger.info("Created at {} - {}", updated.getId(), updated.getTitle());` to BlogService.java code blocks of step 2 of "1. Implement server-side" instead of deleting the log.

Reply:
I also didn't know that we don't log in the service. I'm fine with removing it.